page-promote.php

```diff
@@ -30,7 +30,7 @@
     </div>
     <p>Nextcloud users and enthusiasts can discuss and share Nextcloud experiences on our Forums and IRC channels.</p>
         <ul>
-            <li>Join the <a href="https://help.nextcloud.com">Nextcloud forums</a> and our <a class="hyperlink" href="https://riot.im/app/#/room/#freenode_#nextcloud-dev:matrix.org" target="_blank">chat channel #nextcloud-dev (IRC)</a>, also accessible via <a class="hyperlink" href="https://webchat.freenode.net/?channels=nextcloud-dev" target="_blank">webchat</a>.
+            <li>Join the <a href="https://help.nextcloud.com">Nextcloud forums</a> and our <a class="hyperlink" href="https://app.element.io/#/room/#freenode_#nextcloud-dev:matrix.org" target="_blank">chat channel #nextcloud-dev (IRC)</a>, also accessible via <a class="hyperlink" href="https://webchat.freenode.net/?channels=nextcloud-dev" target="_blank">webchat</a>.
```

Comment:
```suggestion
            <li>Join the <a href="https://help.nextcloud.com">Nextcloud forums</a> and our <a class="hyperlink" href="https://cloud.nextcloud.com/call/xs25tz5y" target="_blank">community chat channel on Talk.</a>
```

The Talk channel is mostly for dev and enthusiast folks, 100% not for support (the forums are for that) but I think 'promo' counts as enthusiasts ;-)